server.js

```diff
@@ -2,6 +2,15 @@ const express = require('express');
 const ws = require('ws');
 const app = express();
 const port = process.env.PORT || 3000;
+const url = require('url');
+
+function fullUrl(req) {
+  return url.format({
+    protocol: req.protocol,
+    host: req.get('host'),
+    pathname: req.originalUrl
+  });
```

Comment:
this returns full url that was requested (we need that to see if the protocol was http or https)